versions/3.0.3.md

```diff
@@ -427,14 +429,16 @@ servers:
 
 #### <a name="serverVariableObject"></a>Server Variable Object
 
-An object representing a Server Variable for server URL template substitution.
+An object representing a Server Variable for URL template substitution in a containing [Server Object](#serverObject).
+
+To form an addressable server URL, the API consumer MUST substitute a valid value for each variable expression in the Server Object's `url` template. The API consumer MAY assign Server Variable values from user input, configuration settings, or any other source. If the API consumer does not have an assigned value for a given server variable, it MUST substitute the provided `default` value.
 
 ##### Fixed Fields
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="serverVariableEnum"></a>enum | [`string`] | An enumeration of string values to be used if the substitution options are from a limited set. The array SHOULD NOT be empty.
-<a name="serverVariableDefault"></a>default | `string` |  **REQUIRED**. The default value to use for substitution, which SHALL be sent if an alternate value is _not_ supplied. Note this behavior is different than the [Schema Object's](#schemaObject) treatment of default values, because in those cases parameter values are optional. If the [`enum`](#serverVariableEnum) is defined, the value SHOULD exist in the enum's values.
+<a name="serverVariableEnum"></a>enum | [`string`] | Optionally specifies an enumeration of valid string values for this variable. The array SHOULD NOT be empty. If `enum` is specified, the API consumer MUST assign one of enumerated values to the server variable.
+<a name="serverVariableDefault"></a>default | `string` |  **REQUIRED**. The default value for the server variable.  If [`enum`](#serverVariableEnum) is defined, the `default` value SHOULD exist in the enum's values.  If the API consumer does not have an assigned alternative value, it MUST substitute the `default` value.  Note that this behavior is different from the [Schema Object's](#schemaObject) treatment of default values. Schema defaults can be resolved by the server, and MAY be omitted from the request; but Server Variables MUST be resolved to an assigned or default value by the client, to assemble a final URL for the API request.
```

Comment:
I translated SHALL in the earlier revision to MUST. Is SHOULD more appropriate here?